packages/cubejs-playground/src/components/RollupDesigner/RollupDesigner.tsx

```diff
@@ -285,7 +285,7 @@ export function RollupDesigner({
           . Please consider removing not additive measures like `countDistinct`
           or `avg`. You can also try to use{' '}
           <Link
-            href="https://cube.dev/docs/schema/reference/pre-aggregations#parameters-type-originalsql"
+            href="https://cube.dev/docs/schema/reference/pre-aggregations#type-originalsql"
```

Comment:
This href doesn't work.

Reply:
Yep, looks like all H4 tags broke when we moved to MDX, thanks for catching it ❤️